go.mod

```diff
@@ -6,38 +6,38 @@ require (
 	github.com/hannahhoward/cbor-gen-for v0.0.0-20200817222906-ea96cece81f1
 	github.com/hannahhoward/go-pubsub v0.0.0-20200423002714-8d62886cc36e
 	github.com/ipfs/go-block-format v0.0.3
-	github.com/ipfs/go-blockservice v0.1.3
+	github.com/ipfs/go-blockservice v0.2.1
 	github.com/ipfs/go-cid v0.0.7
-	github.com/ipfs/go-datastore v0.4.5
-	github.com/ipfs/go-ds-badger v0.2.6
-	github.com/ipfs/go-ipfs-blockstore v0.1.4
+	github.com/ipfs/go-datastore v0.5.0
+	github.com/ipfs/go-ds-badger v0.3.0
+	github.com/ipfs/go-ipfs-blockstore v0.2.1
 	github.com/ipfs/go-ipfs-blocksutil v0.0.1
 	github.com/ipfs/go-ipfs-chunker v0.0.5
 	github.com/ipfs/go-ipfs-delay v0.0.1
-	github.com/ipfs/go-ipfs-exchange-offline v0.0.1
+	github.com/ipfs/go-ipfs-exchange-offline v0.1.1
 	github.com/ipfs/go-ipfs-files v0.0.8
 	github.com/ipfs/go-ipfs-pq v0.0.2
-	github.com/ipfs/go-ipfs-routing v0.1.0
+	github.com/ipfs/go-ipfs-routing v0.2.1
 	github.com/ipfs/go-ipfs-util v0.0.2
 	github.com/ipfs/go-ipld-format v0.2.0
-	github.com/ipfs/go-log/v2 v2.1.1
-	github.com/ipfs/go-merkledag v0.3.2
-	github.com/ipfs/go-peertaskqueue v0.6.0
+	github.com/ipfs/go-log/v2 v2.3.0
+	github.com/ipfs/go-merkledag v0.5.1
+	github.com/ipfs/go-peertaskqueue v0.7.0
 	github.com/ipfs/go-unixfs v0.2.4
 	github.com/ipld/go-codec-dagpb v1.3.0
 	github.com/ipld/go-ipld-prime v0.12.3
 	github.com/jbenet/go-random v0.0.0-20190219211222-123a90aedc0c
 	github.com/libp2p/go-buffer-pool v0.0.2
-	github.com/libp2p/go-libp2p v0.13.0
-	github.com/libp2p/go-libp2p-core v0.8.5
+	github.com/libp2p/go-libp2p v0.16.0-dev.0.20211116110622-8df360043e98
```

Comment:
This is waiting on a go-libp2p release.

Many of these modules have a dependency on go-datastore. While it might be possible to leave go-libp2p a bit lower here due to the tests not exercising the datastore via libp2p it could cause upstream problems, so realistically this is a minimum dependency.

Reply:
double checking -- should we wait for the release to merge @aschmahmann ?

Reply:
Yep. I wouldn't want people relying on the cutting edge go-graphsync to be on an unreleased go-libp2p, that seems like a recipe for false reports.

Hopefully the go-libp2p release can get out in the next couple of days and we can merge this.

Reply:
awesome thanks for clarification.

Reply:
go-libp2p v0.16.0 has been released